commercetools-internal-processors/src/test/resources/io/sphere/sdk/annotations/processors/generators/examples/ChangeQuantity.java.expected

```diff
@@ -0,0 +1,32 @@
+package io.sphere.sdk.annotations.processors.generators.examples.commands.updateactions;
+
+import io.sphere.sdk.annotations.processors.generators.examples.MyResource;
+import io.sphere.sdk.commands.UpdateActionImpl;
+import java.lang.Integer;
+import javax.annotation.Generated;
+
+/**
+ * Updates quantity to MyResource
+ *
+ * {@doc.gen intro}
+ */
+@Generated(
+    value = "io.sphere.sdk.annotations.processors.generators.UpdateActionsGenerator",
+    comments = "Generated from: io.sphere.sdk.annotations.processors.generators.examples.MyResource"
+)
+public final class ChangeQuantity extends UpdateActionImpl<MyResource> {
+  private final Integer quantity;
+
+  private ChangeQuantity(final Integer quantity) {
+    super("changeQuantity");
+    this.quantity = quantity;
+  }
+
+  public Integer getQuantity() {
+    return quantity;
+  }
+
+  public static ChangeQuantity of(final Integer quantity) {
```

Comment:
Here it would be nice to add some javadoc which links to the corresponding get method of the resource:
```
 * @param quantity the quantity {@link MyResource#getquantity()}
```
This will give our users more context how this update action relates to the resource.